### DISCUSSION TITLE
Can the viewer be embedded in a Blazor app?

### DISCUSSION THREAD

**Discussion by MomboMan** [QuestPDF/QuestPDF#409] [Q&A] [answered]
Hi,

Can the PDF viewer be embedded or built into a Blazor app?  All of the ones I'm using are pretty flakey unless I want to spend $$$$

Thanks

Craig

**Comment by bennetbo** [comment, marked as answer]
The QuestPDF previewer doesnt actually display a PDF, it displays images generated by QuestPDF.
The previewer is using Avalonia to run as a standalone app.
Its just not designed for other usage (at least currently) and i dont think it would be desirable either.

**Reply by MomboMan** [reply]
Understood, thanks.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
